src/notebooks_service.py

```diff
@@ -199,33 +282,25 @@ def launch_notebook(user, namespace, project, commit_sha, notebook=None):
         headers=headers,
     )
 
-    # 2. redirect to the server when ready
+    # 2. check response, we expect:
+    #   - HTTP 201 if the server is already running; in this case redirect to it
+    #   - HTTP 202 if the server is spawning
     if r.status_code == 201:
-        # server is already running
+        app.logger.debug(
+            'server {server_name} running'.format(server_name=server_name)
+        )
         return redirect(notebook_url)
```

Comment:
we don't really need to return here since `get_user_server_status` will redirect anyway?